### ISSUE TITLE
New patch release

### ISSUE THREAD

**Issue by fridex** [thoth-station/user-api#1023]
Hey, Kebechet!

Create a new patch release, please.

**Comment by goern** [comment]
/open
/bug
/assign @fridex 

container image build failed:
```
Collecting thoth-common==0.17.0
  Downloading thoth_common-0.17.0-py3-none-any.whl (36 kB)
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    thoth-common==0.17.0 from https://files.pythonhosted.org/packages/c0/10/e163a789a7de2ffd699abc9735d7dfe1249be501e4b6f1ed5902c7b46700/thoth_common-0.17.0-py3-none-any.whl#sha256=e7d517654e42dd0c29a1d9af51fcdb0d7708d206768c4ec3dfbcd582c0b2e7cc (from -r /tmp/requirements_micropipenv-ygl6dhqi.txt (line 2)):
        Expected sha256 08f5942d54e85cce69067f6e4b3f07f079ef273b2b09ae8df711d8228ec56b2b
        Expected     or 7ee03535ccc369d822817c9ce62c7b3ffee37632fd10425d07f646012d7ee60d
             Got        e7d517654e42dd0c29a1d9af51fcdb0d7708d206768c4ec3dfbcd582c0b2e7cc
```

**Comment by fridex** [comment]
> /open
> /bug
> /assign @fridex
> 
> container image build failed:
> 
> ```
> Collecting thoth-common==0.17.0
>   Downloading thoth_common-0.17.0-py3-none-any.whl (36 kB)
> ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
>     thoth-common==0.17.0 from https://files.pythonhosted.org/packages/c0/10/e163a789a7de2ffd699abc9735d7dfe1249be501e4b6f1ed5902c7b46700/thoth_common-0.17.0-py3-none-any.whl#sha256=e7d517654e42dd0c29a1d9af51fcdb0d7708d206768c4ec3dfbcd582c0b2e7cc (from -r /tmp/requirements_micropipenv-ygl6dhqi.txt (line 2)):
>         Expected sha256 08f5942d54e85cce69067f6e4b3f07f079ef273b2b09ae8df711d8228ec56b2b
>         Expected     or 7ee03535ccc369d822817c9ce62c7b3ffee37632fd10425d07f646012d7ee60d
>              Got        e7d517654e42dd0c29a1d9af51fcdb0d7708d206768c4ec3dfbcd582c0b2e7cc
> ```

Is this still valid? I see
```
* :arrow_up: relocked
```

in https://github.com/thoth-station/user-api/pull/1030 so I suppose it has been fixed in v0.6.14 which is [available on quay](https://quay.io/repository/thoth-station/user-api?tab=tags). Please reopen if this is still valid.
